components/homeComponents/KeynoteSpeaker.tsx

```diff
@@ -25,7 +25,7 @@ export default function KeynoteSpeaker(props) {
   }, []);
 
   return (
-    <div className="group flex flex-col items-center w-72 sm:w-72 sm:h-80  rounded-xl shadow-xl relative transition duration-500 ease-in-out overflow-hidden bg-secondary text-white">
```

Comment:
might have conflicts with #213

Reply:
we will deal with it when one of the PRs gets merged; this is not the right place to talk about your own PR